whisper.h

```diff
@@ -147,7 +147,8 @@ extern "C" {
               struct whisper_state * state,
                        const float * samples,
                                int   n_samples,
-                               int   n_threads);
+                               int   n_threads,
+                               bool  debug);
```

Comment:
The `debug` argument cannot stay in the public API.  Same for `whisper_pcm_to_mel_phase_vocoder_with_state()`.

It can remain in `whisper_full_params`